lib/connect.ts

```diff
@@ -1098,7 +1098,7 @@ regular).
             trigger_values: params.trigger_values,
             last_price: parseFloat(params.last_price)
         }
-        let orders = []
+        let orders = [] as any[];
```

Comment:
instead of `any[]` pls user like this `let orders: string[] = []` if values are in array.